File: C/stopwatch.c
#include <stdio.h>

int main()
{   
    int stops;
    scanf("%d", &stops);
    int start_time = 0;
    int stopped_time = 0;
    int total_time = 0;
    int time;
    
    for(int i = 0; i < stops; i++){
        scanf("%d", &time);
        
        if(i%2 == 0){
            start_time = time;
            stopped_time = 0;
        }
        else{
            stopped_time = 1;
            total_time += time - start_time;
        }
        
    }
    
    if(stopped_time == 0){
        printf("still running");
    }
    else{
        printf("%d", total_time);
    }
        
    
    
    return 0;
}

File: C#/stopwatch.cs
using System.IO;
using System;

class Program
{   
    static void Main()
    {
        string line;
        line = Console.ReadLine();
        long stops = Int64.Parse(line);
        
        if(stops % 2 == 1){
            Console.WriteLine("Still running");
        }
        else{
            bool start = true;
            long previousValue = 0;
            long totalTime = 0;
            
            while(stops > 0){
                line = Console.ReadLine();
                long currentValue = Int64.Parse(line);
                   
                if(start){
                    previousValue = currentValue;
                    start = false;
                }
                else{
                    totalTime += currentValue - previousValue;                    
                    start = true;
                }
                
                stops --;
            }
        
            Console.WriteLine(totalTime);
        }
    }
}
